docs/.vitepress/utils/term-parser.ts

```diff
@@ -0,0 +1,39 @@
+export default [
+  {
+    regex: /:([\w+]*):([\w+]*):/g,
+    modifier: (match: RegExpMatchArray) => {
+      console.log(match)
+      const m1 = match[1].replace(/_/g, " ");
+      const m2 = match[2].replace(/_/g, " ");
+      return `<Term term="${m1}" show="${m2}"/>`;
+    },
+  },
+  // Currently duplicating to fix a where bug.
```

Comment:
To fix a what?

Either way, when we have our own `Term` tag, we can use that directly in the future instead of doing string replacements, which is less magical and wouldn't require these parsing rules. Then we also don't need to worry about this weird bug.

Of course, parsing is fine for the moment until all references are migrated, but I'd like to mention somewhere that this is only temporary.

Reply:
Remove the duplicated instance and it will show clearly the bug. For some reason, it seems to skip over some instances. The pageI put in the comment will show this clearly. 

However, I guess replacing these string replacements with our own component would fix it anyway.